lxd/images.go

```diff
@@ -247,7 +248,14 @@ func imgPostContInfo(d *Daemon, r *http.Request, req imagePostReq,
 	tarfile.Close()
 
 	var compressedPath string
-	compress := daemonConfig["images.compression_algorithm"].Get()
+	var compress string
+
+	if compression_algorithm, ok := req.Source["compression"]; ok {
```

Comment:
Can we rename the property from "compression" to "compression_algorithm" for consistency?